crates/swc_css_minifier/tests/fixture/compress-calc/discard-zero/input.css

```diff
@@ -33,3 +33,7 @@
 .class9 {
     width: calc( 0px - (var(--foo, 4px) / 2));
 }
+
+.class10 {
+    width: calc(0 - (pi * 3px) - 0px - 4em);
+}
```

Comment:
I think `calc(0 - 0px)` is an invaild value.

Reply:
You're right, we should have something like `calc(0q - (pi * 3px) - 0px - 4em)`